integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/clouderamanager/action/ClouderaManagerClientActions.java

```diff
@@ -0,0 +1,188 @@
+package com.sequenceiq.it.cloudbreak.util.clouderamanager.action;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.cloudera.api.swagger.RoleConfigGroupsResourceApi;
+import com.cloudera.api.swagger.UsersResourceApi;
+import com.cloudera.api.swagger.client.ApiClient;
+import com.cloudera.api.swagger.client.ApiException;
+import com.cloudera.api.swagger.model.ApiAuthRoleRef;
+import com.cloudera.api.swagger.model.ApiConfigList;
+import com.cloudera.api.swagger.model.ApiUser2;
+import com.cloudera.api.swagger.model.ApiUser2List;
+import com.google.common.base.Strings;
+import com.sequenceiq.it.cloudbreak.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.SdxClient;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.dto.sdx.SdxInternalTestDto;
+import com.sequenceiq.it.cloudbreak.dto.stack.StackTestDto;
+import com.sequenceiq.it.cloudbreak.exception.TestFailException;
+import com.sequenceiq.it.cloudbreak.util.clouderamanager.client.ClouderaManagerClient;
+
+@Component
+public class ClouderaManagerClientActions extends ClouderaManagerClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClouderaManagerClientActions.class);
+
+    private static final String API_ROOT = "/api";
+
+    private static final String API_V_43 = API_ROOT + "/v43";
+
+    @Value("${integrationtest.clouderamanager.defaultUser:}")
+    private String cmUser;
+
+    @Value("${integrationtest.clouderamanager.defaultPassword:}")
+    private String cmPassword;
+
+    @Value("${integrationtest.cloudProvider:}")
+    private String cloudProvider;
+
+    public StackTestDto checkCmTestUser(TestContext testContext, StackTestDto stackTestDto, CloudbreakClient cloudbreakClient) {
```

Comment:
is this needed?

Reply:
Yep, the https://github.com/hortonworks/cloudbreak/blob/dae4debd77cf45027ba34b41761958ec5a91ea11/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/smoke/YarnSmokeTest.java#L81 uses it through the https://github.com/hortonworks/cloudbreak/blob/dae4debd77cf45027ba34b41761958ec5a91ea11/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/clouderamanager/ClouderaManagerUtil.java#L25 reference. I've just updated the related function and did not rewrite it.

Should I rewrite/refactor? How about?

Reply:
can you check if we actually use YarnSmokeTest or not? I found a testsuit yaml for that but I am not sure if we are still using it, the test itself was updated only once in 2020, not sure if it is used anymore